utils/dataconvert/dataconvert.cpp

```diff
@@ -304,11 +304,17 @@ void number_int_value(const string& data,
             {
                 intVal = MIN_TINYINT;
                 pushwarning = true;
+
+                if (saturate)
```

Comment:
I am curios if there are reasons why we can't move the boilerplate code outside the switch/case block?

Reply:
We need to set the flag to true if the value falls outside the min/max range for a given data type, and we only do that check inside the switch case block.